### PR TITLE
Feat/creating administrative module

### PR DESCRIPTION
**Pull request by Felyppe1**
I created the base of the administrative module with the main pages (Upload medias, Edit metadata, Review medias, See medias from my curadoria and My medias) through class based views and styled them (not finished yet).
I also added new fields to the Media model and created a new model called Curadoria. 
Some signals were created for basic functionalities of the administrative module.

### CODE REVIEW

File: meta/admin.py [hunk 1]
```diff
@@ -3,13 +3,22 @@
 from meta.models import *
 from modeltranslation.admin import TranslationAdmin
 
+class CuradoriaAdmin(admin.ModelAdmin):
```
Comment (RianPolonini) [inline meta/admin.py:6]: Wouldn't `CuratorshipAdmin` be better?

File: meta/migrations/0069_media_author_media_curadoria_media_file_media_status.py
Comment (RianPolonini) [inline meta/migrations/0069_media_author_media_curadoria_media_file_media_status.py:1]: Shouldn't the `help_text` be in English to follow the pattern of previous migrations?

File: meta/migrations/0070_alter_media_pub_date_alter_media_timestamp.py
Comment (RianPolonini) [inline meta/migrations/0070_alter_media_pub_date_alter_media_timestamp.py:1]: Shouldn't the `help_text` be in English to follow the pattern of previous migrations?

File: meta/migrations/0071_alter_media_coverpath_alter_media_sitepath.py
Comment (RianPolonini) [inline meta/migrations/0071_alter_media_coverpath_alter_media_sitepath.py:1]: Shouldn't the `help_text` be in English to follow the pattern of previous migrations?

File: meta/migrations/0072_alter_media_filepath.py
Comment (RianPolonini) [inline meta/migrations/0072_alter_media_filepath.py:1]: Shouldn't the `help_text` be in English to follow the pattern of previous migrations?

**Review by RianPolonini — CHANGES_REQUESTED** [review]

File: meta/admin.py (hunk 1 above)
Comment (Felyppe1) [reply meta/admin.py:6]: Yes, we will change it in the next PR. This PR is just for the other developers to have access to the code

File: meta/migrations/0069_media_author_media_curadoria_media_file_media_status.py
Comment (Felyppe1) [reply meta/migrations/0069_media_author_media_curadoria_media_file_media_status.py:1]: I didn't get it. The other ones are in Portuguese as well.

**Pull request merged**